restapi.go

```diff
@@ -826,16 +841,20 @@ func (s *Session) GuildMemberDeleteWithReason(guildID, userID, reason string) (e
 	return
 }
 
-// GuildMemberEdit edits the roles of a member.
-// guildID  : The ID of a Guild.
-// userID   : The ID of a User.
-// roles    : A list of role ID's to set on the member.
-func (s *Session) GuildMemberEdit(guildID, userID string, roles []string) (err error) {
-
-	data := struct {
-		Roles []string `json:"roles"`
-	}{roles}
+// GuildMemberEditData holds all optional parameters for the GuildMemberEdit function.
+type GuildMemberEditData struct {
```

Comment:
The convention in the library for these types of structs is simply `GuildMemberEdit` (see `ChannelEdit`, `MessageEdit`, etc.)